### PR TITLE
feat: PaCh paper

### PR DESCRIPTION
**Pull request by Kohulan**

### CODE REVIEW

File: .DS_Store
Comment (kjappelbaum) [inline .DS_Store:1]: I'd probably delete those - perhaps they are not in our `gitignore`

File: papers/.DS_Store
Comment (kjappelbaum) [inline papers/.DS_Store:1]: I'd probably delete those - perhaps they are not in our `gitignore`

**Comment by kjappelbaum** [comment]
I think it also did not render because it was not included in the index, I'll fix with a separate PR

**Comment by kjappelbaum** [comment]
Thanks for adding it here

**Comment by Kohulan** [comment]
> I think it also did not render because it was not included in the index, I'll fix with a separate PR

Thanks a lot for doing it. Once it is done we can do this in future pull request from our side

**Pull request merged**